WriteVHDLSyntax.py

```diff
@@ -439,6 +439,18 @@ def writeTopLevelMemoryType(mtypeB, memList, memInfo, extraports, delay = 0, spl
 
     first_merge_streamer = True
 
+    first_tp=""
+
+    for memmod in memList:
+        nmem = 0
+        mem=memmod.inst
+        if "L1L2" in mem:
+            first_tp = "TP_L1L2A"
+
+    #If we did not find L1L2 assume L5L^ for reduced project
```

Comment:
@aryd Please fix the typo in this comment:
```suggestion
    # If we did not find L1L2 assume L5L6 for reduced project
```

Reply:
Fixed.